src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
-
+import java.util.Random;
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int randomIndex = new Random().nextInt(Color.values().length);
```

Comment:
You should avoid magic numbers in your code. In this case, `Color.values().length` could be extracted as a constant field to improve code readability and maintainability.